mrb_config.rb

```diff
@@ -113,7 +113,12 @@
     l.library_paths << File.expand_path('soil', bvd)
     l.library_paths.uniq!
 
-    l.libraries << 'glfw'
+    unless Platform.darwin?
+      l.libraries << 'glfw' 
```

Comment:
Ah sorry, your code is fine, @IceDragon200 just fixed travis in master branch. If you rebase against master, it should be fixed